src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -74,15 +75,29 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
     }
 
     private _getParamsScript(initializationData: PaypalCommerceInitializationData, cart: Cart): PaypalCommerceScriptParams {
-        const { clientId, intent, isPayPalCreditAvailable, merchantId, attributionId } = initializationData;
+        const {
+            clientId,
+            intent,
+            isPayPalCreditAvailable,
+            merchantId,
+            attributionId,
+            availableAlternativePaymentMethods = [],
+            enabledAlternativePaymentMethods = [],
+        } = initializationData;
 
-        const disableFunding: DisableFundingType = [ 'card' ];
-        const enableFunding: EnableFundingType = [];
+        const disableFunding: FundingType = [ 'card' ];
+        const enableFunding: FundingType = enabledAlternativePaymentMethods.slice(0);
 
         /**
          *  The default value is different depending on the countries,
-         *  therefore there's a need to add credit, paylater to enable/disable funding explicitly
+         *  therefore there's a need to add credit, paylater or APM name to enable/disable funding explicitly
          */
+        availableAlternativePaymentMethods.forEach(apm => {
+            if (!includes(enabledAlternativePaymentMethods, apm)) {
```

Comment:
Is it possible to use the native `array.includes()` function instead of using lodash includes?

Reply:
@mjhmatt checkout-sdk uses inludes from lodash. I guess because of browser support. ES inludes doesn't work in IE browser